Add proper #includes so that we do not get undef errors to S_ISCHR
during linking.

Signed-off-by: Alex Chiang <achiang@canonical.com>
---
--- zvbi-0.2.33.orig/src/io-dvb.c
+++ zvbi-0.2.33/src/io-dvb.c
@@ -29,6 +29,8 @@
 #include <errno.h>
 #include <sys/select.h>
 #include <sys/ioctl.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 
 #ifndef HAVE_S64_U64
   /* Linux 2.6.x asm/types.h defines __s64 and __u64 only
--- zvbi-0.2.33.orig/src/io-v4l.c
+++ zvbi-0.2.33/src/io-v4l.c
@@ -42,6 +42,7 @@ static const char rcsid [] =
 #include <assert.h>
 #include <sys/time.h>		/* timeval */
 #include <sys/types.h>		/* fd_set, uid_t */
+#include <sys/stat.h>		/* S_ISCHR */
 #include <sys/ioctl.h>		/* for (_)videodev.h */
 #include <pthread.h>
 
--- zvbi-0.2.33.orig/contrib/ntsc-cc.c
+++ zvbi-0.2.33/contrib/ntsc-cc.c
@@ -34,6 +34,7 @@
 #include <locale.h>
 #include <sys/ioctl.h>
 #include <sys/types.h>
+#include <sys/stat.h>
 #include <sys/time.h>
 #ifdef HAVE_GETOPT_LONG
 #  include <getopt.h>
